requirements-dev.txt

```diff
@@ -16,6 +16,7 @@ aioresponses==0.6.0
 moto==1.3.8
 fakeredis==1.0.3
 six>=1.12.0   # upstream - should be removed if fakeredis depends on at least 1.12.0
+mongomock==3.18.0
```

Comment:
it says python 3.3. 3.4, 3.5 (https://pypi.org/project/mongomock/) 🙄 Can we just monkeypatch the crucial parts (should be fine for everything except the insert case).

Reply:
they're testing it on all python versions including 3.7 and 3.8 though - so the pypi info just doesn't seem to be accurate. see https://github.com/mongomock/mongomock/blob/develop/.travis.yml#L17

also monkeypatching the insert and retrieve somewhat defeats the purpose of the test 😄

Reply:
True 👍 Well, it's to build back in case it get's stale. And they probably don't have to update the library often cause the mongo interface rarely changes. Ok then 👍